chia/simulator/block_tools.py

```diff
@@ -388,7 +396,7 @@ async def new_plot(
         exclude_plots: bool = False,
         plot_size: int = 20,
         bitfield: bool = True,
-    ) -> Optional[bytes32]:
+    ) -> tuple[bytes32, bool]:
```

Comment:
Add a dataclass instead of introducing tupleness.

Reply:
its only used in 2 places, is this necessary?

Reply:
i mean i might as well, not worth the discussion.